app/src/main/res/values/strings.xml

```diff
@@ -592,6 +592,7 @@
     <string name="choose_instance_prompt">Choose an instance</string>
     <string name="app_language_title">App language</string>
     <string name="systems_language">System default</string>
+    <string name="remove_watched">Remove Watched</string>
```

Comment:
```suggestion
    <string name="remove_watched">Remove watched</string>
```

Reply:
Fixed in commit [fb193ce](https://github.com/TeamNewPipe/NewPipe/pull/3065/commits/fb193ce3622f34b212bff6a645887f98940eb63c)